src/core/services/layout/layout.js

```diff
@@ -367,6 +375,30 @@
   }
 
   /**
+   * Centralize warnings for known flexbox issues (especially IE-related issues)
+   */
+  function validateAttributeUsage(className, attr, element, $log){
```

Comment:
My worry is that many developers do not read the documentation details.
This is a nice backup for the dev console.